spec/dishes_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'dishes'
+
+describe Dishes do
+
+
+  it { is_expected.to respond_to(:menu)}
```

Comment:
This weekend i learned about the [have_attributes matcher](https://relishapp.com/rspec/rspec-expectations/v/3-8/docs/built-in-matchers/have-attributes-matcher), check it out! How could you use this to achieve the same/a similar test?